elections.md

```diff
@@ -101,4 +91,16 @@ treated as “brand free” activities.
 ## Emeritus Term
 
 Members of the steering committee will graduate to becoming Emeritus members of
-the steering committee.
\ No newline at end of file
+the steering committee.
+
+## Steering committee and election officer recusal
+
+Steering committee members and the appointed election offers pledge to recuse
+themselves from any form of electioneering, including campaigning, nominating,
+endorsing, or even asking people to run.
```

Comment:
"even asking people to run"?  That seems far too extreme.  Also likely to hurt diversity on the committee, since many URM candidates may need an invitation in order to feel like it's worth running.

Reply:
Suggest instead that SC members be allowed to nominate and ask people to run, but not campaign for candidates.

Reply:
Yes, I think that's fair.

Reply:
And I also think that SC members should be allowed to endorse nominees, in the sense that endorsement implies "Yes, I think that this candidate should be allowed to run for office, and I'm prepared to put my name to that".  "Campaigning" I understand to mean actively encouraging people to vote for a particular candidate.  I'm OK if that's not allowed.